Sources/Sentry/Public/SentrySamplingContext.h

```diff
@@ -0,0 +1,31 @@
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@class SentryTransactionContext;
```

Comment:
is this correct? shouldn't be `SentrySamplingContext`?

Reply:
This is a declaration that the SentryTransactionContext class existis without the need to import the actual header.